File: problemas/1000/1035.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = int.parse(entrada[0]);
  final b = int.parse(entrada[1]);
  final c = int.parse(entrada[2]);
  final d = int.parse(entrada[3]);

  if (b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0)
  {
    stdout.writeln('Valores aceitos');
  }
  else
  {
    stdout.writeln('Valores nao aceitos');
  }
}

File: problemas/1000/1035.js
var entrada = require('fs').readFileSync('/dev/stdin', 'utf-8').split(' ');
var [a, b, c, d] = entrada.map(x => Number(x));

if (b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0) {
  console.log('Valores aceitos');
} else {
  console.log('Valores nao aceitos');
}